tools/hack/create-cluster.sh

```diff
@@ -3,10 +3,8 @@
 set -euo pipefail
 
 ## Create kind cluster.
-KIND_NODE_IMAGE=${KIND_NODE_IMAGE:-"docker.io/kindest/node:v1.24.0@sha256:0866296e693efe1fed79d5e6c7af8df71fc73ae45e3679af05342239cdc5bc8e"}
-kind create cluster \
-    --name envoy-gateway-conformance \
-    --image "${KIND_NODE_IMAGE}"
```

Comment:
I think we'll probably want to reintroduce the node image var down the road so we can test against multiple Kubernetes versions, but I'm OK with this for now until we're further along.